py-polars/pyproject.toml

```diff
@@ -40,10 +40,12 @@ Changelog = "https://github.com/pola-rs/polars/releases"
 [project.optional-dependencies]
 # NOTE: keep this list in sync with show_versions() and requirements-dev.txt
 adbc = ["adbc_driver_manager", "adbc_driver_sqlite"]
-async = ["nest_asyncio"]
+async = ["gevent", "nest_asyncio"]
 cloudpickle = ["cloudpickle"]
 connectorx = ["connectorx >= 0.3.2"]
+database = ["adbc_driver_manager", "adbc_driver_sqlite", "connectorx >= 0.3.2", "nest_asyncio", "pandas", "sqlalchemy"]
```

Comment:
Shall we define the 'groups' at the bottom, e.g. first define all 'singular' dependencies alphabetically, then blank line, then groups?